components/dashboard/src/components/IdeOptions.tsx

```diff
@@ -225,8 +248,11 @@ const IdeOptionSwitch = ({
             ))}
         </select>
     );
+    if (versionSelector && hidePinEditorInputs) {
+        versionSelector = <span>{pinnedIdeVersion}</span>;
+    }
```

Comment:
let use a different variable, it's confusing that a selector is not and actual selector